test/modules/TestResultCoordinator/Settings.cs

```diff
@@ -166,7 +166,7 @@ static Settings Create()
                 configuration.GetValue<ushort>("UNMATCHED_RESULTS_MAX_SIZE", DefaultUnmatchedResultsMaxSize),
                 configuration.GetValue<string>("TEST_INFO"),
                 configuration.GetValue("testMode", TestMode.Connectivity),
-                configuration.GetValue("unmatchedResultTolerance", TimeSpan.FromMinutes(1)),
+                configuration.GetValue("unmatchedResultTolerance", TimeSpan.FromMinutes(3)),
                 configuration.GetValue("eventHubDelayTolerance", TimeSpan.FromHours(1)));
         }
 
```

Comment:
Here is the direct method tolerance I was talking about earlier that was rather big:
https://github.com/Azure/iotedge/blob/master/test/modules/TestResultCoordinator/Reports/DirectMethod/Connectivity/DirectMethodConnectivityReport.cs#L98

This failure doesn't seem to happen every time, but theoretically could be caused by a too low `unmatchedResultTolerance`. Should we eliminate this large tolerance I linked above?

Reply:
Thanks for the comments! I think so. Let me also add some logs so if we go over tolerance we will know by how much